spacy/errors.py

```diff
@@ -887,6 +887,7 @@ class Errors(metaclass=ErrorsWithCodes):
     E1021 = ("`pos` value \"{pp}\" is not a valid Universal Dependencies tag. "
              "Non-UD tags should use the `tag` property.")
     E1022 = ("Words must be of type str or int, but input is of type '{wtype}'")
+    E1023 = ("Couldn't read EntityRuler from the {path}. This {frmt} file doesn't exist.")
```

Comment:
Actually, we probably don't even need this parameter:
```suggestion
    E1023 = ("Couldn't read EntityRuler from {path}: the file doesn't exist.")
```

Reply:
OK, I killed this param.